src/api2/managers/wallets-manager.js

```diff
@@ -102,10 +102,11 @@ export class WalletsManager {
    * @param {string} email User's email.
    * @param {string} password User's password.
    * @param {Array} [signers] array of {@link Signer}
+   * @param {Array} [keypairsToSave] array of {@link Keypair} or strings(secret seed) which will be saved to key storage
    *
    * @return {Promise.<object>} User's wallet.
    */
-  async createWithSigners (email, password, signers = []) {
+  async createWithSigners (email, password, signers = [], keypairsToSave = []) {
```

Comment:
keypairs to save where? what is their purpose? mb smth like `additionalKeypairs` fits better